app/views/layouts/_doctype.html.haml

```diff
@@ -0,0 +1 @@
+!!! 5
```

Comment:
but why something obscure like `!!! 5` instead of something like `<!DOCTYPE html>`